### DISCUSSION TITLE
有些软件及网站 网络连接失败

### DISCUSSION THREAD

**Discussion by hao-888** [kkkgo/PaoPaoGateWay#66] [问和答]
有个疑问，我在手机装了一个app，在paopao的dns和网关所构建的网络环境内，这个app会出现网络连接失败，如果切换为手机流量，这个app则正常，我应该怎么排查是哪一个域名或者请求导致，有应该怎么处理这个请求或者域名，才能让这个app在这套网络环境内完美运行

**Comment by kkkgo** [comment]
1、你可以把AUTO_FORWARD设置为no，这样默认不会自动分流，如果这样使用正常，那就是分流问题；
2、分流问题你可以使用ppgw的net_rec功能，观察这个app连了什么域名；
3、你可以自行执行test.sh和debug.sh看看dns是不是正常运作

**Reply by hao-888** [reply]
<img width="272" alt="image" src="https://github.com/kkkgo/PaoPaoGateWay/assets/64783863/d9f97d52-655f-4161-9b9c-f5bcc06f0403">

**Reply by kkkgo** [reply]
需要debug.sh才能看到详细信息，另外你可以升级一下镜像。
